node-fs-writefile/cp.js

```diff
@@ -0,0 +1,9 @@
+
+const note = process.argv[2];
+const fs = require('fs');
+fs.readFile(note, 'utf-8', (err, data) => {
+  if (err) throw err;
+  fs.writeFile('./reminder.txt', `${data}\n`, err => {
+    if (err) throw err;
+  });
+});
```

Comment:
The target filename should be any that the user provides:

---

![image](https://user-images.githubusercontent.com/16143771/159051206-fdb1d537-cacb-48a6-99bb-a516c91bf82f.png)